src/Graphs/IDirectedAcyclicGraph.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Bearded.Utilities.Graphs
+{
+    public interface IDirectedAcyclicGraph<T> : IDirectedGraph<T> where T : IEquatable<T>
+    {
+        IDirectedAcyclicGraph<T> GetTransitiveReduction();
```

Comment:
Should reasonably complicated logic like this be part of the data structure itself? Is this a property of the data or something the data can do?

Reply:
Removed and made the transitive reducer public. I guess we can still make it an extension method and have the same effect though...

Reply:
You say removed, but this is still here. Not sure I follow. 🤔

Reply:
Fixed